packages/xsnap/src/xsnap.js

```diff
@@ -114,7 +114,7 @@ export function xsnap(options) {
     stdio: ['ignore', stdout, stderr, 'pipe', 'pipe'],
   });
 
-  xsnapProcess.on('exit', (code, signal) => {
+  xsnapProcess.once('exit', (code, signal) => {
```

Comment:
is this just hygiene, or is it possible for processes to `.on('exit')` multiple times?

Reply:
It's not possible for `'exit'` to be triggered multiple times (however the base EventEmitter logic doesn't know that), and hygiene dictates that events should be removed once no longer used.